config/prod.yaml

```
@@ -0,0 +1,176 @@
+# Friday AI Assistant - Production Configuration
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)